manifests/g/Geeks3D/MSIKombustor/2/2.6.0.0/Geeks3D.MSIKombustor.2.locale.en-US.yaml

```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Geeks3D.MSIKombustor.2
+PackageVersion: 2.6.0.0
+PackageLocale: en-US
+Publisher: MSI Co., LTD
+PublisherUrl: https://www.geeks3d.com/
+PackageName: MSI Kombustor 2
+PackageUrl: https://geeks3d.com/furmark/kombustor/
+License: Freeware
+Copyright: Copyright ©2010-2015 MSI Co., LTD

```

Comment:
Is the publisher MSI or Geeks3D?
From the about page in geeks3d.com, it seems more like they are a blogger that aggregated multiple apps in their site than them being the actual publisher.

Reply:
From what I know. Geeks3D does have some of their own software, and MSI Kombustor is based of their own [FurMark](https://www.geeks3d.com/furmark/downloads/) benchmarking software, so this software is built jointly by Geeks3D and MSI. I couldn't find download for MSI Kombustor anywhere on MSI's site only a reference to it in a blog.

[MSI Afterburner Now Includes Exclusive Kombustor One-Touch Burn-in for Benchmarking DX11 Game Effect](https://www.msi.com/news/detail/MSIAfterburnerNowIncludesExclusiveKombustorOneTouchBurninforBenchmarkingDX11GameEffect1345)

Reply:
I'm still not convinced that Geeks3D is the publisher. Even if it is, I don't think we should have the package ID as Geeks3D.MSIKombustor, while the publisher and Copyright say MSI. I'll ask the rest of the team to see if what they think.

Reply:
The package ID should be something like `MSI.MSIKombustor` (use MSI instead of Geeks3D). But beyond that, I really think we need a source from MSI for the installer URL.

Reply:
I'll move it to `MSI.Kombustor.4` but MSI doesn't host download for Kombustor.